README.md

```diff
@@ -11,4 +11,6 @@ This is a YouTube Channel Tracker application.
 
 ## Starting the Dev server
  - `cd client`
- - `npm run dev`
\ No newline at end of file
+ - `npm run dev`
+
+Backend api is described in the docstring in server/app/routes.py
```

Comment:
Nitpick but let's add a
`## API Reference` title like we do for starting the dev server